api/src/wfl/executor.clj

```diff
@@ -100,7 +100,7 @@
     (throw (UserException. "Only Dockstore methods are supported."
                            {:status 400 :methodRepoMethod methodRepoMethod}))))
 
-(defn verify-terra-executor
+(defn validate-terra-executor-request
```

Comment:
I like this naming better.

Reply:
Actually, I wouldn't mind consistent naming of all the validate request methods, here's what they look like right now:

- validate-terra-executor-request
- terra-workspace-validate-request-or-throw
- datarepo-sink-validate-request-or-throw
- validate-datarepo-source!
- validate-tdr-snapshot-list

If I were to choose one naming convention, I'd pick `<type>-validate-request-or-throw` or similar.

Reply:
sure - sounds like a plan

Reply:
+1